scanamo/src/main/scala/org/scanamo/request/ScanamoRequest.scala

```diff
@@ -72,3 +73,28 @@ case class RequestCondition(
   attributeNames: Map[String, String],
   dynamoValues: Option[DynamoObject]
 )
+
+case class TransactPutItem(
+  tableName: String,
+  item: DynamoValue,
+  condition: Option[RequestCondition]
```

Comment:
Not yet used, but I've left them here should future work want to implement adding the condition expressions